### DISCUSSION TITLE
突破紀錄，新型量子阱太陽能電池效率近 40%

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#638] [General]
https://technews.tw/2022/05/23/quantum-well-solar-cell/

(雖然跟程式語言沒關係，但我想放出來跟大家科普、分享一下 XD)
太陽能電池基本產生電的材料為PN半導體，一般常見的太陽能電池依照半導體晶格排列分成單晶、多晶、非晶的太陽能電池。這些電池最高的發電效率為25%。而本篇文章發明一種太陽能電池以磷化鎵銦、砷化鎵、砷化鎵銦三種材料製作，產生的發電效率高達39%。

**Comment by stan890425** [comment]
但此種太陽能電池有重金屬的疑慮><

**Comment by sys511613** [comment]
這個轉換效率直接是原本的1.5倍，進步好多
希望能早日解決成本太昂貴的問題

**Comment by jcpan821** [comment]
重點是廢棄品所造成的污染很可怕

**Comment by alexthebigpp** [comment]
製造出太陽能目前來講還是會製造出很多污染，感覺如果能夠大幅提升效率才比較好。

**Comment by yenwei12138** [comment]
期待未來降低成本後能夠拓展到更多的領域應用~

**Comment by Ben6uo** [comment]
成本是一個很好的問題

**Comment by etin2468** [comment]
記得化學系也有在做太陽能電池的研發，不知道效率如何

**Comment by Lynn12378** [comment]
太陽能板的轉換效率大幅提升確實很好，但可能也要注意材料後續廢棄的處理。
